src/scripts/navbar.js

```diff
@@ -0,0 +1,28 @@
+window.onscroll = function() {
+    scrollFunction();
+    stickyFunction();
+};
+
+let navbar = document.getElementById("navbar");
```

Comment:
@Ragnarrlothbrok You can use `const` instead of `let` here if this doesn't change.